active/PagerControl/PagerControl.md

```diff
@@ -0,0 +1,312 @@
+    static Windows.UI.Xaml.DependencyProperty NextButtonVisibilityProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty LastButtonVisibilityProperty{ get; };
+
+    static Windows.UI.Xaml.DependencyProperty FirstButtonCommandProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty PreviousButtonCommandProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty NextButtonCommandProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty LastButtonCommandProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty PagerInputCommandProperty{ get; };
+    
+    static Windows.UI.Xaml.DependencyProperty FirstButtonStyleProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty PreviousButtonStyleProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty NextButtonStyleProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty LastButtonStyleProperty{ get; };
+
+    static Windows.UI.Xaml.DependencyProperty EllipsisShowFirstAndLastProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty SelectedIndexProperty{get; };
+    
+    static Windows.UI.Xaml.DependencyProperty PrefixTextProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty SuffixTextProperty{ get; };
+}
+```
+
+## Appendix
+<!-- Anything else that you want to write down for posterity, but 
+that isn't necessary to understand the purpose and usage of the API.
+For example, implementation details. -->
+
+Here are some end user scenarios for why a developer would want to choose one display mode over the others: 
+
+Button Panel
+* Max wants to search a clothing store's inventory to find a new pair of blue tennis shoes. He searches for blue tennis shoes on the site and is presented with the list of search results. He then scans through the images and descriptions on the first page and does not find anything he likes. He then navigates to the next page and continues searching until he finds the blue tennis shoes he is looking for. 
+
+Combo Box/Number Box
+* Jill opens an application that displays the produce inventory for her grocery store. She sorts her list of results alphabetically and wants to navigate to the page that has the results starting with "L". She knows that what she is looking for is on one of the pages in the middle and wants to navigate there from the list of pages. From there, she can go to the next or previous page to find where the items starting with "L" are. 
+
+Any Display Mode
+* Samanatha opens up an application for the first time and is presented with a tutorial window to show her how to use the application. She needs to move through each page of the tutorial sequentially in order to continue using the application. 
+
+## Accessibility
+
+### UI Automation Patterns 
+* ComboBox and NumberBox will use the control patterns already assigned to them. The pager control will use the selection provider pattern and the individual buttons will use selection item provider. 
```

Comment:
Will we use selection provider for all three kinds (ComboBox, NumberBox and buttons)? How would selection through UIA work then?

Reply:
I am going back and forth between using invoke or selection for the control but you bring up a good point about selection and the other UIA patterns for ComboBox and NumberBox and the UIA patterns they already have. Since everything outside of NumberBox and ComboBox is a button it might make more sense to follow the invoke pattern.